### PR TITLE
added requirements.txt and updated readme

### PR DESCRIPTION
**Pull request by bobby-palko**
Locked dependency versions in a `requirements.txt` file and updated README with new build instructions regarding it

### CODE REVIEW

**Review by ccho-0508 — CHANGES_REQUESTED** [review]
On my macbook the command 'source ./venv/bin/activate' doesn't work - my terminal just prints the message/error "Badly placed ()'s."

I tested it on Windows and it works when I use the 'python' and 'pip' command (not python3 and pip3). @bobrien922 if you could try on your laptop that would be great.

**Comment by ccho-0508** [comment]
I just tried the command 'source ./venv/bin/activate.csh' and that worked!

File: README.md [hunk 1]
```diff
@@ -1,40 +1,56 @@
...
+## How to build
+
+_Note: Visual Studio Code is the recommended editor for the Library Room Auto-Scheduler_
+
+- Python 3.8 or higher is recommended to contribute
+- Clone this repo
+- (Recommended) Create a virtual environment
+  - In the main folder of this project, execute `python3 -m venv venv` on the command line to create the virtual environment
+  - Activate it!
+    - Mac/Linux: `source ./venv/bin/activate`
```
Comment (ccho-0508) [inline README.md:29]: Change to `source ./venv/bin/activate.csh`

Comment (bobby-palko) [reply README.md:29]: @ccho-0508 are you using the C shell? That's the only reason you would have to use the one with the `.csh` extension. I'll update the README to include both

Comment (ccho-0508) [reply README.md:29]: looks like that was the case >:(

**Comment by ccho-0508** [comment]
looks good!

**Comment by ccho-0508** [comment]
FUCK wrong button

**Review by ccho-0508 — APPROVED** [review]
looks good

**Pull request merged**